docs/ab-testing.md

```diff
@@ -136,6 +136,12 @@ can follow our standard [contributing guidelines](https://github.com/guardian/fr
 
 You can stop and start the test using our [switchboard](https://frontend.gutools.co.uk/dev/switchboard).
 
+To see the test in action locally: `localhost:9000/[articleId]#ab-[testName]=[variantName]`
+
+To see the tests you are part of: open the Dev Console -> Resources -> Local Storage -> choose `http://localhost:9000` -> check the `gu.ab.participations` row.
```

Comment:
This is really useful regardless of whether you are on `localhost`, or the `theguardian.com`. Could we update this to clarify that this works on all stages (PROD, CODE and local) of the site?